Jenkinsfile

```
@@ -18,6 +18,7 @@ pipeline {
         ENVIRONMENT_ID = "lab-results-build"
         ECR_REPO_DIR = "lab-results"
         DOCKER_IMAGE = "${DOCKER_REGISTRY}/${ECR_REPO_DIR}:${BUILD_TAG}"
+        AWS_DEPLOY_NON_MAIN_BRANCH = false // false: Skip AWS deployment for all branches other than 'main' ; true: Allow AWS Deployment for all branches
```

This property name doesn't read right for me, just a thought:
```suggestion
        AWS_DEPLOY_ALL_BRANCHES = false // false: Skip AWS deployment for all branches other than 'main' ; true: Allow AWS Deployment for all branches
```

---

Or invert: `AWS_DELPOY_ONLY_MAIN = true`?

---

Variable name changed to AWS_DEPLOY_ONLY_MAIN with inverted logic